src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4617,7 +4620,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 -warnaserror[+|-]             Report all warnings as errors
 -warnaserror[+|-]:&lt;warn list&gt; Report specific warnings as errors
                               (use "nullable" for all nullability warnings)
--warn:&lt;n&gt;                     Set warning level (0-4) (Short form: -w)
+-warnversion:&lt;n&gt;        Enable recently introduced warnings. Default '0'.
```

Comment:
Should we support `-warnversion:?` as we do for `-langversion:?`

Reply:
No plans to do so, since there is not a fixed set of values you can specify.

---
In reply to: [452493597](https://github.com/dotnet/roslyn/pull/45750#discussion_r452493597) [](ancestors = 452493597)